server/views/layout.html

```diff
@@ -17,6 +17,24 @@
   <!--[if lt IE 9]>
     <script src="/html5-shiv/html5shiv.js"></script>
   <![endif]-->
+
+  <!-- Matomo -->
+  <script type="text/javascript">
+    var _paq = window._paq || [];
+    /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+    _paq.push(['trackPageView']);
+    _paq.push(['enableLinkTracking']);
+    (function() {
+      var u="https://fcoforms.matomo.cloud/";
+      _paq.push(['setTrackerUrl', u+'matomo.php']);
+      _paq.push(['setSiteId', '1' ]);
+      _paq.push(['setCustomDimension', '{{ matomoId }}'])
```

Comment:
Is there a reason we'd set the ID as a custom dimension rather than the SiteId?

Reply:
Tried it.. it’s site id that Matomo needs to use. You have to set a custom var/dimension otherwise